index.js

```diff
@@ -0,0 +1,67 @@
+class SuperMath {
+    constructor() {}
+
+    performCalculation(x, y, operator) {
+        switch (operator) {
+            case "+":
+                return x + y;
+            case "-":
+                return x - y;
+            case "*":
+                return x * y;
+            case "/":
+                return x / y;
+            case "%":
+                return x % y;
+            default:
+                throw new Error("Unsupported operator");
+        }
+    }
+
+    check(obj) {
+        const { X, Y, znak } = obj;
+
+        if (["+", "-", "*", "/", "%"].includes(znak)) {
```

Comment:
In fact, you have two validations

Here `["+", "-", "*", "/", "%"].includes(znak)`

And inside the switch you throw error, so, better simplify